examples/relu-metal-cpp/flake.lock

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
No lock files should be commited for the examples, otherwise they'll not use the latest kernel builder.